CHANGES.md

```diff
@@ -3,3 +3,4 @@
 * A new derived table for LDP 1.x, `instance_administrative_notes`,
   extracts administrative notes from instance records.
 
+* Added comments to derived table agreements_package_content_item.
```

Comment:
Maybe specify that this is a Metadb table to avoid any confusion?

Reply:
Done. I don't always think about it. In ERM we only use the Metadb. :-)